tools/scylla-nodetool.cc

```diff
@@ -131,6 +131,31 @@ void cleanup_operation(scylla_rest_client& client, const bpo::variables_map& vm)
     }
 }
 
+void clearsnapshot_operation(scylla_rest_client& client, const bpo::variables_map& vm) {
+    std::unordered_map<sstring, sstring> params;
+
+    if (vm.count("keyspaces")) {
+        std::vector<sstring> keyspaces;
+        const auto all_keyspaces = get_keyspaces(client);
+        for (const auto& keyspace : vm["keyspaces"].as<std::vector<sstring>>()) {
+            if (std::ranges::find(all_keyspaces, keyspace) == all_keyspaces.end()) {
+                throw std::invalid_argument(fmt::format("keyspace {} does not exist", keyspace));
+            }
```

Comment:
In general we should push these checks to the server (which does them anyway). In any case it's okay for now.